test/integration-metrics.test.js

```diff
@@ -139,30 +139,34 @@ describe('Metrics SDK Integration Tests', function () {
   it('should make a request to a Metrics backend with a HAR file', async function () {
     await fetch(`http://localhost:${PORT}`, { method: 'get' });
 
-    const [req, body] = await getPayload();
-    const [har] = body;
```

Comment:
Renamed all these `har` variables because the body here isn't a HAR -- that's under `.request`. Also renamed `getPayload` for the similar reasons.